api/awsauth/awsauth/hubspot_client.py

```diff
@@ -7,7 +8,7 @@ class HubSpotAPICallFailed(Exception):
 
 
 class HubSpotClient:
-    def create_contact(self, email: str) -> dict:
+    def create_contact(self, email: str) -> Optional[dict]:
```

Comment:
I don't see other create_contact calls. Can it be deleted?

Reply:
yeah I'll go ahead and delete - almost went ahead and deleted it